docs/tracing.md

```diff
@@ -86,14 +86,22 @@ config:
   sampler_manager_host_port: ""
   sampler_max_operations: 0
   sampler_refresh_interval: 0s
+  sampler_parent_config:
```

Comment:
I think it would be nice to preface these options with some introductory texts, e.g. what are the defaults used when user does not specify certain options? Maybe we could provide link to OTEL documentation.